pkg/volume/configmap/configmap.go

```diff
@@ -117,13 +117,8 @@ func (plugin *configMapPlugin) NewUnmounter(volName string, podUID types.UID) (v
 }
 
 func (plugin *configMapPlugin) ConstructVolumeSpec(volumeName, mountPath string) (*volume.Spec, error) {
-	configMapVolume := &v1.Volume{
-		Name: volumeName,
-		VolumeSource: v1.VolumeSource{
-			ConfigMap: &v1.ConfigMapVolumeSource{},
-		},
-	}
-	return volume.NewSpecFromVolume(configMapVolume), nil
+	//  To reconstruct volume spec we need source name which cannot be retrieved from volumeName and mountPath.
```

Comment:
What is `source name`, exactly?

It's not possible to reconstruct because v1.ConfigMapVolumeSource 
fields can't be filled with only the params the function receives (`volumeName, mountPath`)?
https://github.com/kubernetes/kubernetes/blob/a0844c17bfb964e3d1709784203e8404f790116c/pkg/apis/core/types.go#L1519-L1540